.gitignore

```diff
@@ -3,3 +3,6 @@ lndr-server.config
 
 # Referenced in docker-compose.yml for persistence between sessions
 /.docker/
+/lndr-backend/data/lndr-server.dev.config
+lndr-server
+.stack-work/
```

Comment:
Less concerned about this line tha nI am about the previous one, but still, as a matter of course, let's restrict this ignore rule to just the root directory by preceding it, too, with a `/`.

Reply:
There are multiple .stack-work directories, should I ignore each one individually?